target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -81,20 +81,25 @@ endef
 TARGET_DEVICES += ubnt_unifi
 
 define Device/ubnt-unifiac
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
-  DEVICE_PROFILE := UBNT
+  ATH_SOC := qca9563
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-ath10k ath10k-firmware-qca988x
   IMAGE_SIZE := 7744k
   IMAGES := sysupgrade.bin
-  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
 
 define Device/ubnt-unifiac-lite
   $(Device/ubnt-unifiac)
   DEVICE_TITLE := Ubiquiti UniFi AC-Lite
   DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
```

Comment:
Why not remove this and do a `DEVICE_PACKAGES += kmod-usb-core kmod-usb2` on Device/ubnt-unifiac-pro

Reply:
yeah, I guess is better that way